src/app/qgsmeasuredialog.h

```diff
@@ -69,6 +69,12 @@ class APP_EXPORT QgsMeasureDialog : public QDialog, private Ui::QgsMeasureBase
     //! When any external settings change
     void updateSettings();
 
+    //! Show the center point X & Y values in line edits when measuring radius
```

Comment:
```suggestion
    //! Show the center point X and Y values in line edits when measuring radius
```

Everywhere, replace '&' by 'and'